demo_nodes_cpp/src/timers/one_off_timer.cpp

```diff
@@ -24,7 +24,7 @@ class OneOffTimerNode : public rclcpp::Node
 {
 public:
   OneOffTimerNode()
-  : rclcpp::Node("one_off_timer"), count(0)
+  : Node("one_off_timer"), count(0)
```

Comment:
This is fine (correct) as a shortcut, but the existing code is also fine and a bit more explicit. I don't think you need to change this for this pr, but I wouldn't necessarily say we need to go out and do a similar change everywhere. Both are correct, and I think it's fine to leave it up to the author to decide which to use in the situation.

Reply:
OK, no worries, I'll go easy on these ones :smile: thanks